packages/typescript/index.js

```diff
@@ -2,6 +2,8 @@ const fs = require('fs')
 const { join } = require('path')
 const basic = require('@antfu/eslint-config-basic')
 
+const tsconfig = process.env['TSCONFIG'] ? process.env['TSCONFIG'] : 'tsconfig.eslint.json'
```

Comment:
```suggestion
const tsconfig = process.env['ESLINT_TSCONFIG'] ? process.env['ESLINT_TSCONFIG'] : 'tsconfig.eslint.json'
```

Maybe this to be more explicit?

Reply:
Good idea. But will `TSCONFIG_FOR_ESLINT`be better?

Reply:
I guess using prefixes would be better for grouping related things by scope. But if you prefer maybe we could support both, like:

```ts
process.env['ESLINT_TSCONFIG'] || process.env['TSCONFIG_ESLINT'] || 'tsconfig.eslint.json'
```

Reply:
Done.